deployments/examples/preemption/yunikorn-configs.yaml

```diff
@@ -0,0 +1,63 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: yunikorn-configs
+  namespace: yunikorn
+data:
+  queues.yaml: |
+    partitions:
+      - name: default
+        queues:
+        - name: root
+          queues:
+          - name: sandbox
+            resources:
+              max:
+                {memory: 1000Mi, vcore: 1000m}
```

Comment:
Yes, it is. Without max resources setting, the available resource depends on users' nodes and it will be hard to determine pod resource requirement.

Reply:
Thanks @chia7712 and @FrankYang0529 's review.
Yes, the max resource is required, otherwise the preemption won't be triggered if the queues have enough resources.

Reply:
Got it and thanks for sharing. I will test it manually later.